builder.go

```diff
@@ -132,8 +116,7 @@ func (b *Bob) maybeQueue(path string) {
 		// we couldn't find the file
 		// most likely a deletion
 		delete(mtimes, path)
-		b.execute()
-		return
+		return true
 	}
 
 	mtime := stat.ModTime()
```

Comment:
Do we still need to calculate the modification time of the files with this new library?